infrastructure/merklemountainrange/tests/support/hashvalues.rs

```diff
@@ -22,32 +22,76 @@
 
 // This file only contains hashvalue lookups for the mmr tests. The values are stored in an array representing the same
 // storage of the mmr. All values are Hex encoded
-use blake2::Blake2b;
 use digest::Digest;
-use std::{fmt::Write, num::ParseIntError};
+use std::fmt::Write;
 
+// this struct is used to contain already computed hashes used by blake2b
```

Comment:
This should all be in a test module so that it is not compiled in unless you are testing. This can probably go in the unit test file that is going to use it?